core/controllers/editor_test.py

```diff
@@ -301,8 +301,8 @@ def _create_training_data(*arg):
                 'change_list': [{
                     'cmd': exp_domain.CMD_EDIT_STATE_PROPERTY,
                     'state_name': state_name,
-                    'property_name': (
-                        exp_domain.STATE_PROPERTY_INTERACTION_UNCLASSIFIED_ANSWERS),
+                    'property_name': (exp_domain.\
```

Comment:
We can't do this; we never use backslashes at the ends of lines.

Could we change the name of the constant throughout the codebase to STATE_PROPERTY_UNCLASSIFIED_ANSWERS?